Pd-C Catalyst/3.names

```diff
@@ -30,4 +30,44 @@ Column 2: Cell voltage (V)
 Column 3: Power density (mW/cm2)
 Column 4: Weight percent of SBH (%) 
 Column 5: Pd/C anode catalyst loading (mg/cm2)
-Cell polarization curves of various Pd/C loading in DBFCs fueled by solution containing different wt% of SBH
\ No newline at end of file
+Cell polarization curves of various Pd/C loading in DBFCs fueled by solution containing different wt% of SBH
+
+0.05wt%:
+CL(mg/cm2)    NOI
+4.4           0
+8.8           0
+13.4          0
+17.7          16
+
+1wt%:
+CL(mg/cm2)    NOI
+4.4           14
+8.8           15
+13.4          16
+17.7          15
+
+3wt%:
+CL(mg/cm2)    NOI
+4.4           14
+8.8           15
+13.4          16
+17.7          15
+
+5wt%:
+CL(mg/cm2)    NOI
+4.4           15
+8.8           14
+13.4          15
+17.7          31
```

Comment:
@sarminh @sepandhaghighi 
There was a mistake here too I used `grep -o SBH,CL | wc -l` but it counts 0.5,17.7 also in this section so that total number of NOIs were true (number of lines - 1) but I turned it to `grep -o ,SBH,CL | wc -l` and it's now OK.